scripts/shared/unit_test.sh

```diff
@@ -54,7 +54,8 @@ for module in "${modules[@]}"; do
 
         echo "Running tests in ${packages[*]}"
         [ "${ARCH}" == "amd64" ] && race=-race
-        ${GO:-go} test -v ${race} -cover "${packages[@]}" -ginkgo.v -ginkgo.trace -ginkgo.reportPassed -ginkgo.reportFile junit.xml "$@"
+        ${GO:-go} test -v ${race} -coverprofile unit_coverage.out "${packages[@]}" -ginkgo.v -ginkgo.trace -ginkgo.reportPassed -ginkgo.reportFile junit.xml "$@" && \
+        go tool cover -func unit_coverage.out
```

Comment:
In case anyone else wonders: it seems these can’t be combined, `go test -cover …` doesn’t produce the same result as `go test -coverprofile … && go tool cover -func …`.

Reply:
Perhaps it would be good to capture this in a comment on the code?

Reply:
I sent inline docs.